modules/gdnative/gdnative_library_editor_plugin.cpp

```diff
@@ -414,6 +414,8 @@ GDNativeLibraryEditorPlugin::GDNativeLibraryEditorPlugin(EditorNode *p_node) {
 	library_editor->set_custom_minimum_size(Size2(0, 250 * EDSCALE));
 	button = p_node->add_bottom_panel_item(TTR("GDNativeLibrary"), library_editor);
 	button->hide();
+
+	editor = nullptr;
```

Comment:
Shouldn't this go in the header? And `button` and `library_editor` can also be initialized to `nullptr` to be safe, even if they then get assigned values here.